enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -2465,6 +2465,16 @@ func (c *FirecrackerContainer) stopMachine(ctx context.Context) error {
 	if err := c.machine.StopVMM(); err != nil {
 		return status.WrapError(err, "kill firecracker")
 	}
+
```

Comment:
Should this go after the `c.machine.Wait` block, as opposed to in a defer? I'm seeing a couple `wait for firecracker to exit: context deadline exceeded` errors in the logs. In that case, I think this removal code will also fail because there will still be a live process in the cgroup

Reply:
yeah, I guess whether this is deferred or not, we'll still probably have cases where the cgroup might not be cleaned up. Just moved after the `machine.Wait()` for now